opm/parser/eclipse/EclipseState/Schedule/MSW/AICD.hpp

```diff
@@ -54,6 +54,15 @@ namespace Opm {
             AutoICD::serializeOp(serializer);
         }
 
+        const double flowRateExponent() const;
+        const double viscExponent() const;
+        const double oilDensityExponent() const;
+        const double waterDensityExponent() const;
+        const double gasDensityExponent() const;
+        const double oilViscExponent() const;
+        const double waterViscExponent() const;
```

Comment:
No reason to have `const` om value return - caller should be free to do whatever  with the value

Reply:
You are right.